cola/models/prefs.py

```diff
@@ -9,6 +9,7 @@
 from ..cmd import Command
 
 
+AUTOTEMPLATE = "cola.commit.autotemplate"
```

Comment:
Single quotes for consistency, please.  This seems reasonable.    One small note -- everywhere else we use just a single flat namespace for config values.  e.g. `cola.autoloadcommittemplate` is more in-line with the rest of the config keys we use.  In documentation we spell it as `cola.autoloadCommitTemplate` which helps the readability.

Please make sure this feature is documented in `doc/git-cola.rst` in the `CONFIGURATION VARIABLES` section.

Reply:
```suggestion
AUTOTEMPLATE = "cola.commit.autotemplate"
```
I used this format because of your comment (see https://github.com/git-cola/git-cola/issues/735#issuecomment-336750453)

Reply:
Ah, sorry for not remembering.  Thanks for the updates.